src/pages/RequestsPage.js

```diff
@@ -1,45 +1,63 @@
-/* eslint-disable jsx-a11y/img-redundant-alt */
-import { RequestCard } from "@the-statics/shared-components";
+/* eslint-disable no-unused-vars */
+import { RequestCard, Button02 } from "@the-statics/shared-components";
 import styles from "./RequestsPage.module.scss";
+import { useSelector } from "react-redux";
+import { selectUser } from "../features/user/userSlice";
+import { useState } from "react";
+import { useMutation } from "react-query";
+import axios from "axios";
+import ApiService from "../services/Api";
+import { useNavigate } from "react-router-dom";
+
+const ApiInstance = new ApiService(axios);
 
 function RequestsPage() {
+  const navigate = useNavigate();
+  const user = useSelector(selectUser);
+  const [error, setError] = useState("");
+  const onChatStartClick = (id) => () => {
+    const attendants = { attendants: [id, user.id] };
+
+    mutate(attendants);
+  };
```

Comment:
```suggestion
  const navigate = useNavigate();
  
  const user = useSelector(selectUser);
  
  const [error, setError] = useState("");
  
  const onChatStartClick = (requesterId) => () => {
    const attendants = { attendants: [requesterId, user.id] };

    mutate(attendants);
  };
```
가독성을 위해 다른 라이브러리에서 불러온 코드는 한 칸 띄워주시는 것이 좋다고 생각합니다 ㅎㅎ

그리고 requesterId 와 같이 누구와 누구가 채팅에 참가하게 되는지도 명시해주시면 좋을 것 같습니다

Reply:
넵, 완료했습니다~